src/v0/util/index.js

```diff
@@ -47,6 +47,12 @@ const flattenMap = (collection) => _.flatMap(collection, (x) => x);
 // GENERIC UTLITY
 // ========================================================================
 
+const getEndpointPath = (endpoint) => {
+  // extract the path details from the whole endpoint url passed
+  const url = new URL(endpoint);
+  return url.pathname.replace(/^\/+/, '');
```

Comment:
this replaces "/" from url? but there may be dynamic data in the url right like user id or email etc

Reply:
I used URL.pathname to extract whole of the path after the domain. It returns '/' always at start so handled that here. [Ref](https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname)

Reply:
I am replacing the dynamic portions of url with 'xx'. Does this suffice?